src/Router.tsx

```diff
@@ -1,6 +1,10 @@
-import { lazy, Suspense } from "react";
+import React, { lazy, Suspense } from "react";
 import { Outlet, RouteObject, useRoutes, BrowserRouter } from "react-router-dom";
 
+import Sidebar from "components/Sidebar";
+import TeamHistory from "containers/TeamHistory";
```

Comment:
I think Router should only import from `Screens`. The Screen should then import the container that it needs